### ISSUE TITLE
What's the equivalent of GCC's -fpermissive?

### ISSUE THREAD

**Issue by ruanjiashing** [LADSoft/OrangeC#746]
Some code just will not compile without -fpermissive. I don't know the equivalent switch on OCC.

**Comment by GitMensch** [comment]
The question is: which code does not compile with OCC but compiles with `gcc -fpermissive`...

**Comment by ruanjiashing** [comment]
> The question is: which code does not compile with OCC but compiles with `gcc -fpermissive`...

No. This is hiding from the question not the question itself. You give me the equivalent for -fpermissive or simply tell me such a switch exists or not.

**Comment by GitMensch** [comment]
Then I declare "there is no need for a switch like that because occ compiles everything without such a switch"...

**Comment by ruanjiashing** [comment]
> Then I declare "there is no need for a switch like that because **occ compiles everything without such a switch**"...

This is impossible. For some source code you will always need something like -fpermissive :smile:

**Comment by GitMensch** [comment]
Show some code like that and we'll see...

**Comment by ruanjiashing** [comment]
> Show some code like that and we'll see...

I'm trying to get the FOX Toolkit compile: https://github.com/LADSoft/OrangeC/issues/747
